buildkernel

```diff
@@ -1540,8 +1540,19 @@ ensure_efi_partition_mounted() {
     check_key_file_if_on_efi_system_partition
 }
 enter_build_directory() {
-    show "Proceeding - entering ${LINUXDIR}..."
-    cd "${LINUXDIR}"
+    if [ -z ${LINUXBUILDDIR+x} ]; then
+        LINUXBUILDDIR=$LINUXDIR
+    else
+        if [ -d "$LINUXBUILDDIR" ]; then
+            show "Proceeding - Cleaning ${LINUXBUILDDIR}..."
+            rm -r ${LINUXBUILDDIR}
+        fi
+        show "Proceeding - copying from ${LINUXDIR} to ${LINUXBUILDDIR}..."
+        cp -LR ${LINUXDIR} ${LINUXBUILDDIR}
```

Comment:
`cp -LR`? Not sure that's quite what is wanted - it's going to deref any symbolic links in the kernel tree. Perhaps consider `cp -ax` instead (archive copy, [staying on one filesystem](https://unix.stackexchange.com/questions/256704/what-does-cp-x-do/256714#256714) for safety)?